src/client/src/tscSQLParser.c

```diff
@@ -2214,10 +2207,6 @@ int32_t validateSelectNodeList(SSqlCmd* pCmd, SQueryInfo* pQueryInfo, SArray* pS
        hasDistinct = (pItem->distinct == true);
        distIdx     =  hasDistinct ? i : -1;
     }
-    if(pItem->aliasName != NULL && validateColumnName(pItem->aliasName) != TSDB_CODE_SUCCESS){
```

Comment:
Relax restriction on an aliasName. The aliasName itself is not required to be TK_ID or TK_STRING, since the aliasName might be a sequence of Chinese characters, which are not recognizable as token TK_ID or TK_STRING.